app/server/ruby/test/lang/core/test_random.rb

```diff
@@ -1,156 +1,163 @@
-require_relative "../../setup_test"

```

Comment:
The large diff in this file is mostly to do with replacing windows line endings that I accidentally committed last time I edited this file 😅